Assets/XML/GameText/Events_CIV4GameText.xml

```diff
@@ -861,6 +861,14 @@
 		<Russian>Сообщают, что из-за мощных ураганов сильно пострадали рыбаки в державе %s1_civ_adjective.</Russian>
 		<Spanish>Un fuerte temporal ha interrumpido la actividad pesquera en el imperio %s1_civ_adjective.</Spanish>
 	</TEXT>
+    <TEXT>
+        <Tag>TXT_KEY_EVENTTRIGGER_MASSIVE_FOREST_WILDFIRE</Tag>
+        <English>Due to recent droughts a massive wildfire has started near the %s1_civ_adjective city of %s2_city.</English>
+        <French></French>
+        <German></German>
+        <Italian></Italian>
+        <Spanish></Spanish>
```

Comment:
Just delete the empty language lines, here. Doesn't matter much either way, just a subjective preference from me.